Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
     "fraos",
     "indexed-log-map",
     "multiverse",
+    "tools/first-address-from-root",
```

Comment:
let's name it sth like `cli-tools` to be more specific. and the name of the crate somehow related to cardano e.g. `cardano-first-address-from-bip44`

Reply:
well... I guess it's not actually bip44, but cip1852. But good idea.

Reply:
we can do even better: cardano-cli-tools crate with first-address-from-bip44